app/src/main/java/dev/sebastiano/bundel/onboarding/Previews.kt

```diff
@@ -0,0 +1,100 @@
+package dev.sebastiano.bundel.onboarding
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import dev.sebastiano.bundel.ui.BundelYouTheme
+import dev.sebastiano.bundel.ui.composables.TimeRange
+import dev.sebastiano.bundel.ui.composables.TimeRangeRow
+import dev.sebastiano.bundel.ui.composables.checkedMaterialPillAppearance
+import dev.sebastiano.bundel.ui.composables.onboardingCheckedPillAppearance
+import dev.sebastiano.bundel.ui.composables.onboardingUncheckedPillAppearance
+import java.time.LocalTime
+
+@Suppress("unused")
+internal class OnboardingPreviews {
```

Comment:
Should we put these in the `debug` sourceset at this point?

Reply:
that is a lifestyle choice I'll leave for you to decide :-)